assets/src/blocks/CarouselHeader/deprecated/carouselHeaderV1.js

```diff
@@ -15,18 +15,18 @@ export const carouselHeaderV1 = {
           link_text: '',
           link_url: '',
           link_url_new_tab: false,
-        }
+        },
       ],
     },
   },
-  isEligible({ slides }) {
-    return slides?.some(slide => typeof slide.header_size !== 'undefined');
+  isEligible({slides}) {
+    return slides?.some((slide) => typeof slide.header_size !== 'undefined');
```

Comment:
From my point of view, I preferred to use `( )` only if the `function` receives more than 1 `param`.